ChangeLog.md

```diff
@@ -4,21 +4,172 @@ Changelog
+- Made the window resize tab bigger
+- Reworked pedia window layout, putting search bar and navigation buttons at top, similar to a web browser standard layout
+- Fixed display of circle arrow icons on production screen
+- Made options.xml contain only modified options by default, but added a button to output the full options database to the options window
+- Reworked splash / menu screen layout
+- Added time to produce at the currently selected location with available PP to production item tooltips
+- Added an option to reset empire readiness when joining a game, so orders can be revised even after they were submitted
+- Replace total cost and minimum production time with turns to produce, if fully funded, at current production location to the producible items list
+- Added indication in tooltips when production cost or minimum time are location-dependent and no location is selected
+- Added option to disable disclosure of galaxy seed
+- Prohibit typing some characters into galaxy seed text box, to prevent pedia layout script injection
+- The server stop auto-advancing turns after a player has won the game
+- Relabelled Resign button to Exit to Menu
+- Removed unused / redundant keymaps configuration file
+- Repeatedly clicking to close the game window will override the default autosave and exit faster
+
+#### Content and Balance
+
+- Stringtables
+    - Added a few names to lists used for ship names
+    - Translation updates: Spanish
+    - Cleaned up stringtables, removing duplicates from english in other languages
+    - Added stealth and slots to hull descriptions
+    - Fuel efficiency effects are now described like "-40%" instead of like "60%"
+    - Fixed some typos
+    - Corrected colony base hull description about slots
+    - Fixed descriptions of concentration camp effects
+
+- New Stuff
+    - Added Arc Disruptor ship weapon part
+    - Added sitreps for gifting to allied empires
+
+- Reworked what weapons can target fighters, ships, and planets
+    - Mass drivers, Lasers, Plasmas, Death Rays, and Spinal antimatter cannon attack ships and planets (no fighters)
+    - Interceptors attack enemy fighters only
+    - Fighters attack ships and fighters
+    - Bombers attack ships only
+    - Heavy bombers are re-enabled and attack ships and planets
+
+- Rebalanced fighters
+    - Interceptors launch 50% faster than other fighters
+    - Fighter refinement techs increase interceptor capacity and launch rate, not damage
+    - Techs increase strike fighter damage by +2, bomber damage by +3, and heavy bomber damage by +6
+    - Adjusted costs and build times
+    - Species piloting traits give -1 to +3 for all fighters except interceptors
+
+- Ships
+    - Ship production cost increases ("upkeep") also count ships on the production queue
+    - Reduced Shield part costs
+    - Colony base hulls may not mount engines or fuel parts
+    - Set Colony Base hull structure to 2
```

Comment:
base hull is more expensive but has three internal slots

Reply:
you want to modify a line or add a line?